### PR TITLE
Completing translation

### PR DESCRIPTION
**Pull request by DedrickEnc**
The en.json and the fr.json was complete based on the server log

### CODE REVIEW

**Comment by jniles** [comment]
@dedrickenc, this PR breaks the build because of some poorly written tests.

This test: `Account Management creates multiple accounts with the batch option selected`, uses the value 'Recettes', which has been updated.  Could you update this tests to either search through translation keys instead of translated values?

**Comment by DedrickEnc** [comment]
@jniles @sfount

File: client/test/e2e/accounts/accounts.spec.js [hunk 1]
```diff
@@ -92,6 +92,8 @@ describe('Account Management', function () {
       number : parentNumber,
       label : 'End to End Test: '
     };
+    let select = $('body').element(by.model('AccountEditCtrl.account.type_id'));
```
Comment (jniles) [inline client/test/e2e/accounts/accounts.spec.js:95]: Why not just `element(by.model('AccountEdtiCtrl.account.type_id'))`?

**Comment by jniles** [comment]
LGTM if tests pass.

**Pull request merged**